system-tests/projects/plugin-run-events/cypress.config.js

```diff
@@ -26,8 +35,14 @@ module.exports = {
         })
       })
 
-      on('before:spec', (spec) => {
-        console.log('before:spec:', spec.relative)
+      on('before:spec', (spec = {}) => {
```

Comment:
```suggestion
      on('before:spec', (spec) => {
```

Spec should never be null, it even has the correct type definitions (yay!), so this is not necessary.